tests/Model/PHPMethod.php

```diff
@@ -44,10 +44,16 @@ public function readObjectFromReflection($reflectionObject): static
      */
     public function readObjectFromStubNode($node): static
     {
-        $this->parentName = $this->getFQN($node->getAttribute('parent'));
+        $this->parentName = self::getFQN($node->getAttribute('parent'));
         $this->name = $node->name->name;
+        $typeFromAttribute = self::findTypeFromAttribute($node->attrGroups);
         $this->availableVersionsRangeFromAttribute = self::findAvailableVersionsRangeFromAttribute($node->attrGroups);
-        $this->returnType = self::convertParsedTypeToString($node->getReturnType());
+        if ($typeFromAttribute != null) {
+            array_push($this->returnTypesFromSignature, ...array_map(fn(string $type) => preg_replace('/\w+\[]/', 'array', $type),
```

Comment:
This code is duplicated a couple of times. Should we change `findTypeFromAttribute` to return array as well?

Reply:
Yes, thought about it. You confirmed my doubts. Will update this. Thanks!